rest/data/version_test.go

```diff
@@ -230,3 +231,171 @@ func (s *VersionConnectorSuite) TestRestartVersion() {
 		s.Equal("caller3", b1.ActivatedBy)
 	}
 }
+
+func (s *VersionConnectorSuite) TestGetVersionsAndVariants() {
+	if s.isMock { // mock method not implemented
```

Comment:
this makes me go :|

Reply:
I can create a mock if you think it's necessary, but it would just be a mock for contents in the db. It doesn't make testing any more meaningful

Reply:
Oh, sorry i misread that :|. oops.